src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1450,6 +1450,9 @@ private function addHttpClientSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->scalarNode('mock_response_factory')
+                            ->info('ID of the service used to generate mock responses. Must be invokable.')
```

Comment:
The id of the service that should generate mock responses. It should be either an invokable or an iterable.

Reply:
Fixed